doc/tutorials/rtvm_workload_design_guideline.rst

```diff
@@ -0,0 +1,44 @@
+.. _rtvm_workload_guideline:
+
+Guidelines to the workload design in an RTVM
```

Comment:
Could we rephrase this as "Guidelines for designing an application to run in an RTVM"?

@dbkinder thoughts?